site/src/main/mdoc/landing.template.html

```diff
@@ -26,7 +26,7 @@
   <link rel="stylesheet" type="text/css" href="helium/laika-helium.css" />
   <link rel="stylesheet" type="text/css" href="helium/landing.page.css" />
   <link rel="stylesheet" type="text/css" href="styles/landing.page.css" />
-  <script>initVersions("", "", "");</script>
+  <script>initVersions("", "", " ");</script>
```

Comment:
Don't even ask ...

Reply:
### Before
![Screen Shot 2022-03-18 at 9 54 11](https://user-images.githubusercontent.com/3119428/159047895-97628483-011e-4f60-aff6-7cbfad3ebb72.png)

### After
![Screen Shot 2022-03-18 at 9 54 37](https://user-images.githubusercontent.com/3119428/159047898-83cb1c09-9825-427a-b490-05c19c14085e.png)

Reply:
Sorry for being late on this, but why did we choose the `1` over `1.0`? Is the point that we will have pages for major versions only (`1`, `2` and not `1.1`, `1.2`, etc)?

Reply:
No worries, I'd like to hear your opinion on this too :) But yes, exactly. Discussion was in https://github.com/http4s/http4s/pull/6087#issuecomment-1060749888.